config/crd/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -601,8 +601,10 @@ spec:
                   or from a secret with the field ''proxy'''
                 properties:
                   value:
+                    description: nolint:unused
```

Comment:
Please make sure the real description of the property is added here, i guess that the first comment besides property definition in the dynakube_types.go, and is added by controller-gen automatically